upsint/utils.py

```diff
@@ -130,35 +130,20 @@ def prompt_for_pr_content(template: str):
     Create a temporary file and feed it to $EDITOR so the user defines content
     for the PR, mainly title and body.
     """
-    t = tempfile.NamedTemporaryFile(delete=False, prefix="gh.")
-    try:
-        template_b = template.encode("utf-8")
-        t.write(template_b)
-        t.flush()
-        t.close()
-        try:
-            editor_cmdstring = os.environ["EDITOR"]
-        except KeyError:
-            logger.warning("EDITOR environment variable is not set")
-            editor_cmdstring = "/bin/vi"
-
-        logger.debug("using editor: %s", editor_cmdstring)
-
-        cmd = [editor_cmdstring, t.name]
-
-        logger.debug("invoking editor: %s", cmd)
-        proc = subprocess.Popen(cmd)
-        ret = proc.wait()
-        logger.debug("editor returned : %s", ret)
-        if ret:
-            raise RuntimeError("error from editor")
-        with open(t.name) as fd:
-            pr_content = fd.read()
-        if template == pr_content:
-            logger.error("PR description is unchanged")
-            raise RuntimeError("The template is not changed, the PR won't be created.")
-    finally:
-        os.unlink(t.name)
+    if "EDITOR" in os.environ:
```

Comment:
No need to: https://github.com/pallets/click/blob/main/src/click/_termui_impl.py#L467-L479

Reply:
Ah that's convenient! I'll add another commit